src/pages/onboarding/completion/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { textBox } from './index.styles';
+import { Complete } from '@/assets/icons';
+import { useNavigate } from 'react-router-dom';
+import { Text, Button } from '@/components';
+import { useState, useEffect } from 'react';
+import { useGetUserNameQuery } from '@/queries/user/user-name';
+import theme from '@/styles/theme';
+
+export default function Completion() {
+  const navigate = useNavigate();
+  const { data } = useGetUserNameQuery();
+  const [nickName, setNickName] = useState('');
+
+  useEffect(() => {
+    setNickName(data.nickname);
+  }, []);
+  const handleCompleteButtonClick = () => {
+    navigate('/', { replace: true });
+  };
+
+  return (
+    <div css={textBox}>
+      <Complete width="70px" />
+      <Text typo="title1" color={theme.colors.green200}>
+        {nickName} 님<br />
```

Comment:
react query를 사용하는 이유는 서버에서 받은 데이터를 state에 저장하지 않기 위함입니다.
`data`로 response data`를 바로 받을 수 있기 때문에 아래처럼 변경해 주세요
```suggestion
  const handleCompleteButtonClick = () => {
    navigate('/', { replace: true });
  };

  return (
    <div css={textBox}>
      <Complete width="70px" />
      <Text typo="title1" color={theme.colors.green200}>
        {data.nickName} 님<br />
```